### PR TITLE
Update README with rosdep steps:

### PR DESCRIPTION
**Pull request by LanderU**
* Include rosdep steps to install rmw_iceoryx deps
* Fix ROS 2

Signed-off-by: LanderU <lander.usategui@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,13 @@ git clone --branch LATEST_ROS_VERSION https://github.com/ros2/rmw_iceoryx.git
...
 
 rmw_iceoryx is compatible with ROS 2 starting with Eloquent release.
-Assuming you have ROS2 installed correctly, you can compile the iceoryx workspace with colcon:
+Assuming you have ROS 2 installed correctly, you can compile the iceoryx workspace with colcon:
 
 ```bash
 cd ~/iceoryx_ws/
 source /opt/ros/LATEST_ROS_VERSION/setup.bash  # alternatively source your own ROS 2 workspace
+rosdep update
+rosdep install --from-paths src --ignore-src --rosdistro LATEST_ROS_VERSION -y
```
Comment (mossmaurice) [inline README.md:31]: @LanderU Which dependecies will be installed here that were missing for you?

**Review by mossmaurice — COMMENTED** [review]
Thanks for your contribution! LGTM, one question before we go ahead and merge.

Edit: Could you also create a PR that add the same change to `master`? Thanks!

File: README.md (hunk 1 above)
Comment (LanderU) [reply README.md:31]: @mossmaurice, `ros-ROS_DISTRO-test-msgs`

**Comment by LanderU** [comment]
@mossmaurice, I'll open a new PR for master, thank you!

**Review by mossmaurice — APPROVED** [review]

**Pull request merged**